### ISSUE TITLE
Homes from other player not found

### ISSUE THREAD

**Issue by ghost** [masagameplay/MaSuiteHomes#13]
Sometimes it happens that with /homes playername the homes of a user are not displayed although the player has homes in the database. 

It does not matter if the player is online or offline. Even if he is offline and on the same server as I am, the player-not-found message from MaSuite/homes/messages.yml will always appear. 
Even if the player reconnects, I still cannot load his homes. But the player himself can see his homes.

I use the newest dev builds (state 12.05.2020 18:00 o'clock)

**Comment by DakirDEV** [comment]
I found out that this only happens to users who do not have their current player name in the masuite_players table at username because they renamed themselves but this was not updated in the table.

**Comment by masagameplay** [comment]
The username is updated when the player logins to the server. The name updation cannot check automitcally if the player has updated name if they have not logged in.

TL;DR: You need to use the old name if player has changed their name and not logged in.

**Comment by DakirDEV** [comment]
Is there a setting that this is updated at login? Because I have just two users online who still have the old name in the database.

**Comment by chesnutcase** [comment]
Chipping in because I thought this was an issue of playernames vs UUIDs:

> You need to use the old name if player has changed their name and not logged in.

But @ghost says "even if the player reconnects", so maybe just the player logging in is not enough to trigger the update?

**Comment by masagameplay** [comment]
It seems like that I've forgotten to re-add player name updation on the rewrites... Now it should be fixed in [here](https://github.com/masagameplay/MaSuiteCore/commit/cc47df770fdf04f3a0791e7604d874a7101a1d15) and the dev build is coming to [here (#74)](https://ci.codemc.io/job/masagameplay/job/MaSuiteCore/)

**Comment by DakirDEV** [comment]
Many thanks for the quick help

**Comment by DakirDEV** [comment]
Fixed in the new dev build [#74](https://ci.codemc.io/job/masagameplay/job/MaSuiteCore/74/)
Issue can be closed.

**Comment by masagameplay** [comment]
I will close this now but if this issue persists feel free to open this.
